utils/parse_cert_set_result.py

```diff
@@ -0,0 +1,69 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0.
+
+"""
+
+A simply utility script to parse out certificates and keys from some IoT operations.
```

Comment:
should this sit next to the fleet provisioning sample?
I don't know

Reply:
I feel it's more general, it's useful to anyone regardless of the sdk language (assuming they have python installed)